pkgs/by-name/ea/earbuds/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  installShellFiles,
+  pkg-config,
+  dbus,
+  libpulseaudio,
+  bluez,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "earbuds";
+  version = "0.1.9";
+
+  src = fetchFromGitHub {
+    owner = "JojiiOfficial";
+    repo = "LiveBudsCli";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-A4XQiJrk4ehb6+935L2JFOeAhUJ7bdukV5mL0Jxn0sQ=";
+  };
+  cargoHash = "sha256-FFZnYsqoG2Fq1K2D0ElwTPXgp9F1MyBj27EOAALfXaM=";
+
+  nativeBuildInputs = [
+    pkg-config
+    installShellFiles
+  ];
+
+  buildInputs = [
+    dbus
+    libpulseaudio
+    bluez
+  ];
+
+  doCheck = false;
```

Comment:
Please state reason in comment (Also are all tests failing?)

Reply:
> Please state reason in comment (Also are all tests failing?)

there are no unit or integration tests present in the application (?)

Reply:
Annotation, `# no unit or integration tests present`